modules/coo-server-side-apply.adoc

```diff
@@ -0,0 +1,305 @@
+----
+
+.. Apply the YAMl by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f ./prom-spec-edited.yaml --server-side 
+----
++
+[NOTE]
+====
+You must use the `--server-side` flag.
+====
+
+.. Get the changed Prometheus object and note there is one more section in `managedFields` which has `spec.enforcedSampleLimit`:
++
+[source,terminal]
+----
+$ oc get prometheus -n coo-demo 
+----
++
+.Example output
+[source,yaml]
+----
+managedFields: <1>
+- apiVersion: monitoring.rhobs/v1
+  fieldsType: FieldsV1
+  fieldsV1:
+    f:metadata:
+      f:labels:
+        f:app.kubernetes.io/managed-by: {}
+        f:app.kubernetes.io/name: {}
+        f:app.kubernetes.io/part-of: {}
+    f:spec:
+      f:enforcedSampleLimit: {} <2>
+  manager: kubectl
+  operation: Apply
+----
+<1> `managedFields`
+<2> `spec.enforcedSampleLimit`
+
+. Modify a field managed by the `MonitoringStack` resource: 
+.. Change `spec.LogLevel`, a field managed by the `MonitoringStack` resource, using the following YAML configuration:
++
+[source,yaml]
+----
+# changing the logLevel from debug to info
+apiVersion: monitoring.rhobs/v1
+kind: Prometheus
+metadata:
+  name: sample-monitoring-stack
+  namespace: coo-demo
+spec:
+  logLevel: info <1>
+----
+<1> `spec.logLevel` has been added
+
+.. Apply the YAML by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f ./prom-spec-edited.yaml --server-side 
+----
++
+.Example output
++
+[source,terminal]
+----
+error: Apply failed with 1 conflict: conflict with "observability-operator": .spec.logLevel
+Please review the fields above--they currently have other managers. Here
+are the ways you can resolve this warning:
+* If you intend to manage all of these fields, please re-run the apply
+  command with the `--force-conflicts` flag.
+* If you do not intend to manage all of the fields, please edit your
+  manifest to remove references to the fields that should keep their
+  current managers.
+* You may co-own fields by updating your manifest to match the existing
+  value; in this case, you'll become the manager if the other manager(s)
+  stop managing the field (remove it from their configuration).
+See https://kubernetes.io/docs/reference/using-api/server-side-apply/#conflicts
+----
+
+.. Notice that the field `spec.logLevel` cannot be changed using Server-Side Apply, because it is already managed by `observability-operator`.
+
+.. Use the `--force-conflicts` flag to force the change.
++
+[source,terminal]
+----
+$ oc apply -f ./prom-spec-edited.yaml --server-side --force-conflicts 
+----
++
+.Example output
++
+[source,terminal]
+----
+prometheus.monitoring.rhobs/sample-monitoring-stack serverside-applied
+----
++
+With `--force-conflicts` flag, the field can be forced to change, but since the same field is also managed by the `MonitoringStack` resource, the Observability operator detects the change, and reverts it back to the value set by the `MonitoringStack` resource.
```

Comment:
```suggestion
With `--force-conflicts` flag, the field can be forced to change, but since the same field is also managed by the `MonitoringStack` resource, the Observability Operator detects the change, and reverts it back to the value set by the `MonitoringStack` resource.
```